src/states/Game.js

```diff
@@ -1,49 +1,93 @@
 /* globals __DEV__ */
 import Phaser from 'phaser';
-//import Mushroom from '../sprites/Mushroom';
+import StaticAsset from '../sprites/staticAsset';
+import Player from '../sprites/player';
 
 export default class extends Phaser.State {
   init() {}
   preload() {}
 
   create() {
```

Comment:
Throughout this `create()` function, I recommend you add more comments as to what's happening so it's easier for another developer (or you in a few weeks) to understand what the code is doing in the game.